### ISSUE TITLE
Bad galaxy sizes in Herbel example

### ISSUE THREAD

**Issue by rrjbca** [skypyproject/skypy#182]
**Describe the bug**
The Herbel galaxy example is giving physical galaxy sizes of order ~10^5 kpc leading to angular sizes of order ~10 degrees. The coefficients used by the physical size model are taken from https://arxiv.org/abs/2001.07727 but they may have used a different parameterisation of the linear-lognormal size model compared to https://arxiv.org/abs/astro-ph/0301527 which is what we based our model on. It is also not clear if `mu_phys` in the first reference is the same as `Rbar_50` in the second reference so they could be modelling different size measures.

**To Reproduce**
```
skypy -c examples/herbel_galaxies.yaml -f fits
```
```
>>> hdu = fits.open("./blue_galaxies.fits")
>>> np.mean(hdu[1].data['physical_size'])
241537.8853921193
>>> np.mean(hdu[1].data['angular_size'])
0.13882203524226688
```

**Expected behavior**
Figure 4 in https://arxiv.org/abs/astro-ph/0301527 shows that Rbar_50 should be of order 1kpc.

**Comment by rrjbca** [comment]
Changes in #208 to the luminosity function could have changed the sizes in this example so best to check the values again. Other things to check would be the zeropoint and the coefficients for the `physical_size` model

**Comment by philipp128** [comment]
> Changes in #208 to the luminosity function could have changed the sizes in this example so best to check the values again. Other things to check would be the zeropoint and the coefficients for the `physical_size` model

It didn't change. I get this output now:
```
>>>  hdu = fits.open("./blue_galaxies.fits")
>>> np.mean(hdu[1].data['physical_size'])
2910619.451085817
>>> np.mean(hdu[1].data['angular_size'])
0.7330269954807195
```

I also can't see where the absolute magnitude in `herbel_galaxies.yaml` is changed to the r-band to compare it to figure 4 in https://arxiv.org/abs/astro-ph/0301527. But this shouldn't give such a big difference.

**Comment by rrjbca** [comment]
This was resolved by #266 by using the size model parameters from [Shen et al. 2003](https://arxiv.org/abs/astro-ph/0301527). Closing.
